Deploy/stacks/dynamic/common-scripts/build.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
```

Comment:
are changes to this file intended to be committed?

Reply:
Potentially not. Does this cause any issues?

Reply:
It looks like it might clash with the new changes in launch/tasks.json

Reply:
Which new changes? And which clashes are you expecting? So, shall I just revert those changes?

Reply:
I'm not sure actually, are these changes required for python?

Reply:
I don't think you should use the debug compose file for deploying though

Reply:
I am puzzled now. I thought you were referring to having changed the line ending by accident as you only quoted `#!/bin/bash`. However, if you refer to the actual changes in the file, they have been made by George and I have only merged them into my branch. So, I believe they should be compatible with other changes. Plut please confirm with checking the file history. Thanks

Reply:
> I don't think you should use the debug compose file for deploying though

I believe `debug compose` is only included in case a debug port is provided

Reply:
Let's have George approve this

Reply:
Ok I understand the changes now, they're required for python images

Reply:
Yes, this is required so that a separate debug image can be built.